developers.md

```diff
@@ -0,0 +1,81 @@
+#Developers Guide
+
+This guide is intended to help developers understand the underlying concept of MetaStats.
+
+##Architecture
+The architecture of MetaStats is based on Bob Martin's _Clean Architecture_ concepts.
+
+![architecture](doc/architecture_metaStats-updated_arch.png)
```

Comment:
This is not using proper UML arrow heads.

Reply:
Adjusted with 29c21e2